src/Tools/GenerateProps/targetgroups.props

```diff
@@ -74,6 +74,9 @@
     </TargetGroups>
     <TargetGroups Include="netstandard">
       <NuGetTargetMoniker>.NETStandard,Version=v2.0</NuGetTargetMoniker>
+      <!-- By default, netstandard libraries will be held to the UWP compatibility bar;
+           set this to false in the project to force otherwise. -->
+      <EnableWinRT>true</EnableWinRT>
```

Comment:
This is a poor name. It is not enabling WinRT. WinRT is not the same thing as UWP.

A better name may be: `<EnableUWPCompatibilityChecks>`